src/main/java/ChatBot.java

```diff
@@ -0,0 +1,53 @@
+import esme.Ui;
+
+import java.util.Scanner;
+
+public class ChatBot {
+    private Scanner in;
```

Comment:
Consider a more precise variable name for ``Scanner``, such as ``inputScanner``?